book/_toc.yml

```diff
@@ -46,6 +46,11 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+        - file: tutorials/OOP
```

Comment:
jupyterbook expects a file here, so I recommend a simple `tutorials/OOP/index.md` that just has a title: 
```
# Object Oriented Programming
```